autoware_cmake/package.xml

```diff
@@ -9,8 +9,6 @@
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
-  <exec_depend>ros_environment</exec_depend>
```

Comment:
Since it's true that `autoware_cmake` depends on `ros_environment`, I guess we should write the dependency in some way.

Seeing the example of `gtest_vendor` in `ament_cmake_gtest`, is `buildtool_export_depend` the correct tag?
https://github.com/ament/ament_cmake/blob/c5489e8c22489625a804618a160649953d21dec8/ament_cmake_gtest/package.xml#L19
https://github.com/ament/ament_cmake/blob/c5489e8c22489625a804618a160649953d21dec8/ament_cmake_gtest/ament_cmake_gtest-extras.cmake#L41

Is that wrong, too?

Reply:
Oh, https://github.com/autowarefoundation/autoware_common/pull/144 did it. Thank you @jspricke!

Reply:
@esteve Thank you. Hmm, let me confirm a little more.

There are two facts:

1. `ros_environment` provides the environment variables `ROS_VERSION` and `ROS_DISTRO`.

```
share/ros_environment/package.xml
4:  <description>The package provides the environment variables `ROS_VERSION` and `ROS_DISTRO`.</description>
```

2. Several packages, which depend on `ROS_VERSION` (one of the `ros_environment`'s env), depend on `ros_environment`.

https://github.com/tier4/osqp_vendor/blob/1ea3682dacc08bd05d301098b202e74bd6416251/package.xml#L19

What's the difference between those packages and `autoware_cmake`?